unicodetools/src/main/java/org/unicode/xml/UCDPropertyDetail.java

```diff
@@ -287,16 +317,19 @@ public class UCDPropertyDetail {
                     true,
                     false,
                     false,
-                    true);
-    public static UCDPropertyDetail Hyphen_Detail =
-            new UCDPropertyDetail(
-                    UcdProperty.Hyphen,
-                    VersionInfo.getInstance(2, 0, 0),
-                    32,
                     true,
-                    false,
-                    false,
-                    true);
+                    false);
+// Deprecated
+//    public static UCDPropertyDetail Hyphen_Detail =
+//            new UCDPropertyDetail(
+//                    UcdProperty.Hyphen,
+//                    VersionInfo.getInstance(2, 0, 0),
+//                    32,
+//                    true,
+//                    false,
+//                    false,
+//                    true,
+//                    false);
```

Comment:
i suggest you just delete this altogether

Reply:
I screwed up here - I should have added values for maxVersion, which I have done now.